galaxy_ng/app/migrations/0017_populate_repos_and_remotes.py

```diff
@@ -40,6 +40,10 @@
             ),
             "pulp_type": "ansible.ansible",
             "rate_limit": 8,
+        },
+        "distribution": {
+            "name": "rh-certified",
```

Comment:
when I ran this command from the installer: https://github.com/ansible/galaxy_collection/blob/devel/roles/post_install_config/tasks/configure_remotes.yml#L34-L41 it created a distro with `red-hat-certified` for the base_path and name. I wonder if we could make this red-hat-certified as well.